src/DependencyInjection/DoctrineEncryptExtension.php

```diff
@@ -58,9 +59,24 @@ public function load(array $configs, ContainerBuilder $container): void
             $loader->load('services_with_secret.yml');
         }
 
-        // Remove usage of AttributeAnnotationReader when using php < 8.0
-        if (PHP_VERSION_ID < 80000) {
-            $container->setAlias('ambta_doctrine_annotation_reader','annotations.reader');
+        // Symfony 1-4
+        // Sanity-check since this should be blocked by composer.json
+        if (Kernel::MAJOR_VERSION < 5 || (Kernel::MAJOR_VERSION === 5 && Kernel::MINOR_VERSION < 4)) {
+            throw new \RuntimeException('doctrineencryptbundle/doctrine-encrypt-bundle expects symfony-version >= 5.4!');
+        }
+
+        // Symfony 5-6
+        if (Kernel::MAJOR_VERSION < 7) {
```

Comment:
Just tested it with one of my projects where I'm using symfony 6.4, but doctrine-extension 7. 

Sadly this check does not work as expected and might need some adjustments for that case

Reply:
As said in https://github.com/DoctrineEncryptBundle/DoctrineEncryptBundle/pull/30#issuecomment-1984276193, I think this issue should be resolved by sticking to either 6.4 or 7.0, but not both at the same time.